src/v3/src/components/PasswordRequirements/PasswordRequirements.tsx

```diff
@@ -85,9 +88,15 @@ const PasswordRequirements: UISchemaElementComponent<{
   );
 
   useEffect(() => {
-    passwordValidationHandler((password ?? '') as string);
+    const userInfoObj = Object.keys(userInfo).length ? userInfo : userProvidedUserInfo;
```

Comment:
So `userInfo` will be defined in all remediations besides enroll profile?

Reply:
But also, yes, enroll-profile does not provide userInfo because they are essentially creating the user object here, so IDX doesn't have any information to provide since this is the step that creates the user.